third_party/2and3/werkzeug/wrappers.pyi

```diff
@@ -44,8 +49,12 @@ class BaseRequest:
     args: ImmutableMultiDict
     @property
     def data(self) -> bytes: ...
-    # TODO: once Literal types are supported, overload with as_text
-    def get_data(self, cache: bool = ..., as_text: bool = ..., parse_form_data: bool = ...) -> Any: ...  # returns bytes if as_text is False (the default), else Text
+    @overload
+    def get_data(self, cache: bool = ..., as_text: Literal[False] = ..., parse_form_data: bool = ...) -> bytes: ...
+    @overload
+    def get_data(self, cache: bool, as_text: Literal[True], parse_form_data: bool = ...) -> Text: ...
+    @overload
+    def get_data(self, *, as_text: Literal[True], parse_form_data: bool = ...) -> Text: ...
```

Comment:
Same here; we need an overload on `bool` for the case where the type checker can't infer the value.